index.js

```diff
@@ -320,12 +335,14 @@ class KeyringController extends EventEmitter {
       })
   }
 
-  // Sign Message
-  // @object msgParams
-  //
-  // returns Promise(@buffer rawSig)
-  //
-  // Attempts to sign the provided @object msgParams.
+  /**
+   * Sign Message
+   *
+   * Attempts to sign the provided message parameters.
+   *
+   * @param {Object} msgParams - The message parameters to sign.
+   * @returns {Promise<Buffer>} The raw signature, if successful.
```

Comment:
Nit: is there a reason the `, if successful` was added here, but not on other return descriptions? (e.g. such as the one for `signPersonalMessage`. It is sorta implied when returning a Promise that the thing-in-the-promise won't be returned if the Promise is rejected.

Reply:
I was going back and forth re: this. I'm now of the School of Implied Promise Failure.